vendor/github.com/opensds/opensds/contrib/connector/fc/fc.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2018 Huawei Technologies Co., Ltd. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fc
+
+import (
+	"github.com/opensds/opensds/contrib/connector"
+)
+
+const (
+	fcDriver = "fibre_channel"
+	PortName = "port_name"
+	NodeName = "node_name"
+	Wwpn     = "wwpn"
```

Comment:
I suggest moving these constants to `connector.go` so as to avoid importing all protocols module inside `csi/server/server.go`